Locator/FailureTransportLocator.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Locator;
+
+use Symfony\Component\Messenger\Transport\TransportInterface;
+use Symfony\Contracts\Service\ServiceProviderInterface;
+
+class FailureTransportLocator
+{
+    private $receiverLocator;
+    private $failureReceiverName;
+
+    public function __construct(ServiceProviderInterface $receiverLocator, string $failureReceiverName)
+    {
+        $this->receiverLocator = $receiverLocator;
+        $this->failureReceiverName = $failureReceiverName;
+    }
+
+    public function getFailureTransport(): TransportInterface
```

Comment:
i've just realized that it would produce an error if `transport_failure` is null

Reply:
Let's see if we want to merge this into `ReceiverLocator` and then see if this comment still applies. But yea, if no `failure_transport` is selected, then that whole section of the page should just not show up.